api-extractor.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "<projectFolder>/lib/hls.d.ts",
```

Pulled down the PR, ran `npm ci && npm run build` and got:

```
ERROR: Error parsing /Users/rwalch/dev/hls.js/api-extractor.json:
The "mainEntryPointFilePath" path does not exist: ~/dev/hls.js/lib/hls.d.ts
```

Does this it need to point `lib/src/hls.d.ts`?

```suggestion
  "mainEntryPointFilePath": "<projectFolder>/lib/src/hls.d.ts",
```

---

`lib/src/hls.d.ts` doesn't exist for me. It's at lib/hls.d.ts on my machine